producto/posintegrado/README.md

```diff
@@ -0,0 +1,158 @@
+  - Proveedor de Soporte de Sistema.
+  - Acceso Remoto.
+- La caja debe contar con a lo menos un puerto `RS232` o en su defecto con entrada USB exclusiva para el Dispositivo POS.
+- El comercio debe disponer en cada local con puntos exclusivos para conectar a Internet vía Lan `RJ45` los POS.
+- Se recomienda segmentar el ancho de banda con `300 kb` exclusivos para el uso del POS, dado que cuando existen procesos batch, cámaras de seguridad, uso de aplicaciones de escritorio, etc. bajo la mismo segmento pueden afectar una transacción si la banda ancha contratada no es suficientemente alta.
+- El local debe disponer con un punto de corriente exclusivo de `220v` para enchufar el POS.
+
+### Requisitos del Comercio
+
+- El comercio debe contar con una o más áreas encargadas de los siguientes puntos:
+  - Procedimientos de Contingencia ante caídas de señal Internet y energía eléctrica.
+  - Procedimiento de Respaldo y Recuperación de operaciones efectuadas en POS Integrado de Transbank.
+  - Homologación en versiones sistemas de Caja.
+  - Homologación en Sistema Operativos de Caja.
+  - Manejo de Claves Supervisora y de usuarios de Sistema Operativo.
+  - Procedimientos de Seguridad y responsabilidad en el uso de la información proporcionada por Transbank y Clientes.
+  - Capacitación y divulgación de uso de nuevos sistemas.
+  - Manuales de Sistemas.
+  - Soporte de Redes.
+
+<aside class="notice">
+Se recomienda que el Establecimiento posea la misma versión de Sistema Operativo y Software de Caja en todas sus cajas. En caso de poseer distintas versiones, será necesario efectuar un proceso de Integración y Certificación con Transbank por cada uno de ellos.
+</aside>
+
+<aside class="notice">
+Ante cualquier cambio que efectúe el Establecimiento, ya sea de Sistema Operativo o Software de Caja, podrían existir impactos en la Integración efectuada con el POS Integrado, por lo que se sugiere probar la correcta operación con POS Integrado antes de efectuar el cambio en ambiente Productivo.
+</aside>
+
+## Seguridad
+
+### Confidencialidad de la Información
+
+De acuerdo a las normativas vigentes, las transacciones con Tarjeta de Crédito y Débito requieren la
+incorporación de los siguientes elementos de seguridad en el sistema:
+La información leída en el punto de venta NO debe ser almacenada en ningún sistema.
+Para efectos de cuadratura e identificación de transacciones, se debe utilizar el NÚMERO DE OPERACION.
+
+### Tratamiento de los Tracks de la Tarjeta
+
+La información grabada en el Track I y Track II será leída sólo por los dispositivos de seguridad (POS). Estos
+dispositivos procederán a encriptar el contenido del Track I y Track II para que esta información no quede en
+claro.
+
+### Tratamiento de datos sensibles
+
+Para asegurar la confidencialidad de la información, los mensajes de las transacciones o al menos los datos
+sensibles (además del PIN) deben viajar encriptados en los distintos tramos de la conexión, tanto en el
```

Comment:
Isn't that something that the POS does by itself? AFAIU, the integrator doesn't need to anything special for this  to be true. In that case I would change the wording from "deben viajar encriptados" por "viajan encriptados".